activerecord/test/cases/enum_test.rb

```diff
@@ -551,4 +551,13 @@ def self.name; "Book"; end
   test "data type of Enum type" do
     assert_equal :integer, Book.type_for_attribute("status").type
   end
+
+  test "scopes can be disabled" do
+    klass = Class.new(ActiveRecord::Base) do
+      self.table_name = "books"
+      enum status: [:proposed, :written], _scopes: false
+    end
+
+    assert_raises(NoMethodError) { klass.proposed? }
```

Comment:
Shouldn't this be `klass.proposed` ? which is the scope?

Reply:
fixed it.. 😄